src/library/index.ts

```diff
@@ -49,7 +51,9 @@ export default function (options: LibraryOptions): Rule {
                     }),
                     move(projectRoot),
                 ]), MergeStrategy.AllowCreationConflict),
-            addDependenciesToPackageJson()
+            addDependenciesToPackageJson(),
+            modifyWorkspace(tempOptions.name),

```

Comment:
Why use tempOptions here instead of options?

Reply:
Because we want to use the original name here. tempOptions.name = '@coben/common-ui', while options.name = 'common-ui'

Reply:
Introduce a descriptive variable for that, then.